prompting/rewards/multiple_choice.py

```diff
@@ -0,0 +1,52 @@
+import time
+import torch
+import re
+from typing import List
+from prompting.rewards import BaseRewardModel, BatchRewardOutput
+from prompting.dendrite import DendriteResponseEvent
+
+
+class MultipleChoiceModel(BaseRewardModel):
+    @property
+    def name(self) -> str:
+        return "multiple_choice"
+
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.choices = ("A", "B", "C", "D")
```

Comment:
Slightly, but my gut tells me it's more likely out of distribution for the type of task which may bring more pain..

Also, we are taking the **last** occurrence of `{'A','B','C','D'}` which should make the incidental 'A' match condition less likely. 

Maybe a specific delimiter is a better way such as `A)`. There's a tradeoff between robustness and flexibility -- we don't want to throw away good answers because they don't conform